tensorflow/lite/kernels/internal/reference/reference_ops.h

```diff
@@ -2912,6 +2912,24 @@ inline void Cast(const RuntimeShape& input_shape, const SrcT* input_data,
   }
 }
 
+struct FloatMod {
```

Comment:
Why do we need overload this operator? It seems like the original one doesn't have this.

Reply:
@haozha111 , thanks for the  review, the original one also has this. Kindly check and approve.

Reply:
Could you also remove the 'struct FloatMod' from floor_mod.cc?

Reply:
Also I think it's better to keep 'struct FloatMod' local to function 'FloorMod'. Probably you can just use a lambda function to do this (instead of declaring a new struct) ?

Reply:
@haozha111 , thanks for the review, i have shifted the 'struct FloatMod' local to function 'FloorMod'. I tried other methods as well but they were increasing the code size. Hope this is ok. Kindly check and approve.

Regards
Amit